handlers/addautohandler.go

```diff
@@ -17,16 +23,61 @@ func AddAutoHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	defer database.Close()
+	if r.Method == "GET" {
+		log.Printf("D: HTTP GET on /add-auto")
+		htmlHeadPath := "/templates/head.gohtml"
+		navBarPath := "/templates/navbar.gohtml"
+		templatePath := "/templates/addauto.gohtml"
 
-	htmlHeadPath := "/templates/head.gohtml"
-	navBarPath := "/templates/navbar.gohtml"
-	templatePath := "/templates/addauto.gohtml"
+		t, err := template.ParseFiles(templatePath, htmlHeadPath, navBarPath)
+		if err != nil {
+			log.Printf("E: Error processing template")
+		}
+		t.Execute(w, nil)
+	}
+	if r.Method == "POST" {
+		log.Printf("D: HTTP POST on /add-customer")
+		r.ParseForm()
+		var err error
+		var DateOfSale time.Time
```

Comment:
local scope variables begin with a lowercase letter - consider "saleDate" for example